wolox-books/src/views/Register.vue

```diff
@@ -32,10 +32,11 @@
       input.input-field(type='password' v-model='confirmPassword')
       span.error-message(v-if='$v.confirmPassword.$invalid && submitted')
         | password do not match.
-    button.green-button.submit-button(type='submit')
+    button.green-button(type='submit')
       | Sign up
-    button.green-button.login-button
-      | Log in
+    router-link(to='/login')
+      button.grey-button(type='button')
```

Comment:
No need to add the `button` since it's a `router-link` component. Just apply the `.grey-button` styles to the link